client/src/App.js

```diff
@@ -96,9 +97,41 @@ function App() {
             />
           </div>
         </section>
-
-        <section className="app-info"></section>
-        <section className="lightbox w-full justify-center items-center flex">
+         </section>
+        <section className="app-info w-screen h-screen p-8 relative">
+          <div className="halogen-line absolute bottom-0 left-0"></div>
+          <div className="flex h-1/2 w-full">
+               <div className="w-1/2 flex justify-center">
+                 <img 
+                 className="h-full p-8"
+                 src="https://user-images.githubusercontent.com/13723156/127782437-7f221d77-d4d9-4371-ad6d-ede848f03248.png" alt="" /> 
```

Comment:
Let's make sure to add `alt` text here describing the content of the images. Otherwise looks great!!!

Reply:
ok, I made the changes. Ready to merge!